tutorials/v7/ntuple/ntpl003_lhcbOpenData.C

```diff
@@ -54,8 +54,7 @@ void Convert() {
    // We create RNTuple fields based on the types found in the TTree
    // This simple approach only works for trees with simple branches and only one leaf per branch
    auto tree = f->Get<TTree>("DecayTree");
-   std::vector<TBranch*> branches;
-   for (TObject *obj : *(tree->GetListOfBranches())) {
+   for (auto obj : *(tree->GetListOfBranches())) {
```

Comment:
Instead of
```
for (auto obj : *(tree->GetListOfBranches())) {
   auto b = static_cast<TBranch*>(obj); 
```
another option is 
```
for(auto b : TRangeStaticCast<TBranch>( *tree->GetListOfBranches())) {
```

Reply:
I like it but I think it would need to be `ROOT::Detail::TRangeStaticCast`.  Do we generally use `Detail` things in tutorials?

Reply:
Right I forgot, we hid that one ... the one we settle on for a public interface is TRangeDynCast:
```
for(auto b : TRangeDynCast<TBranch>( *tree->GetListOfBranches())) {
```
@Axel-Naumann opinion?

Reply:
`TRangeDynCast` requires the tutorial to do `if (!b) continue;`: we should certainly check the validity of the dynamic cast.